starter-project-backend-g31/src/routes/rating.ts

```diff
@@ -0,0 +1,58 @@
+import express from "express"
+import {
+    getAllRatings,
+    getSingleRating,
+    createRating,
+    updateRating,
+    deleteRating,
+    getAllRatingsForAGivenArticle,
+    getAllRatingsForAGivenArticleAndUser,
+    getAllRatingsForAGivenUser
+} from "../controller/rating"
+
+const ratingRouter = express.Router()
+
+ratingRouter.get("/", async (req, res) => {
+    const response = await getAllRatings();
```

Comment:
better to handle business logic on the controller, and use this file for only routing purposes. 
So that you need to move async(req,res) part to controller, and call that function from here